### PR TITLE
4.5: Added MAVLink instructions to common-efi.rst

### PR DESCRIPTION
**Pull request by Max-001**
Please refer to this commit: https://github.com/ArduPilot/ardupilot/commit/afc019e1eaa116adf68ff0de69902578521d14d6

### CODE REVIEW

File: common/source/docs/common-efi.rst [hunk 1]
```diff
@@ -87,3 +87,12 @@ First, set either RPM1 or RPM2 sensor as being sourced from the EFI (examples wi
 
 - set :ref:`RPM1_TYPE<RPM1_TYPE>` = 3 (EFI)
 - then setup the :ref:`common-rpm-based-notch`
+
+EFI connected via MAVLink
+==============================
+
+Some EFIs do not use the autopilot as their computer, those EFIs have their own onboard computer. It might be possible to connect those EFIs to the autopilot by a MAVLink connection (e.g. then it is possible to read the EFI's status, temperature, etc. by a LUA script). The following parameter should be set to connect an EFI via MAVLink:
```
Comment (Hwurzburg) [inline common/source/docs/common-efi.rst:94]:
```suggestion
Some EFIs connect to the autopilot via MAVLink, instead of by a Serial port or DroneCAN.  The following parameter should be set to connect such an EFI via MAVLink:
```

File: common/source/docs/common-efi.rst [hunk 2]
```diff
@@ -87,3 +87,12 @@ First, set either RPM1 or RPM2 sensor as being sourced from the EFI (examples wi
 
 - set :ref:`RPM1_TYPE<RPM1_TYPE>` = 3 (EFI)
 - then setup the :ref:`common-rpm-based-notch`
+
+EFI connected via MAVLink
+==============================
```
Comment (Hwurzburg) [inline common/source/docs/common-efi.rst:92]: title underline incorrect

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
Thanks for this....a few things:
1. I think this is another connection/setup method and should be under the others above, rather than down here.
2. Title underline is too long...must match text above in length
3. I would reword the statement as shown
unfortunately since you did not  create a local working branch and instead made the changes directly on your master branch, I cannot fix these items for you....unless I created a new PR myself.

**Comment by Max-001** [comment]
Thanks @Hwurzburg for the review! I'm completely new to Ardupilot Wiki, sorry for my mistakes.

I added the suggested changes, please let me know if this is OK?

**Review by Hwurzburg — APPROVED** [review]
lgtm...thanks...will link this in upcoming features

**Comment by Hwurzburg** [comment]
sorry could not resolve conflicts on this so created a new PR and merged it
